scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "lectern-scripts",
-	"version": "2.0.0-beta.5",
+	"version": "2.0.0-beta.6",
```

Comment:
This `scripts` package isn't meant to be imported anywhere... it could maybe have the version removed like the root dir?

Reply:
amending here